components/cosmetic_filters/renderer/cosmetic_filters_js_handler.cc

```diff
@@ -116,6 +116,51 @@ const char kHideSelectorsInjectScript[] =
           };
         })();)";
 
+const char kRemovalsInjectScript[] =
```

Comment:
This script looks sophisticated enough to be moved to a separate file.
At least ide highlighting and code completion will work.

I also suggest to use TS instead JS for it.